docs/qml/qcl.md

```diff
@@ -87,25 +84,27 @@ using a mean-square error loss, Adam optimizer. Training is performend on the GP
 if available:
 
 ```python exec="on" source="material-block" session="qcl" result="json"
-
-# train the model
-n_epochs = 200
-lr = 0.5
+n_epochs = 100
+lr = 0.25
 
 input_values = {"phi": x_train}
 mse_loss = torch.nn.MSELoss()  # standard PyTorch loss function
 optimizer = torch.optim.Adam(model.parameters(), lr=lr)  # standard PyTorch Adam optimizer
 
-print(f"Initial loss: {mse_loss(model(input_values), y_train)}")
-
-y_pred_initial = model({"phi": x_test})
+print(f"Initial loss: {mse_loss(model(values=x_train), y_train)}")
+y_pred_initial = model(values=x_test)
 
 running_loss = 0.0
 for i in range(n_epochs):
 
     optimizer.zero_grad()
 
-    loss = mse_loss(model(input_values), y_train)
+    # given a `n_batch` number of input points and a `n_observables`
+    # number of input observables to measure, the QNN returns
+    # an output of the following shape: [n_batch x n_observables]
+    # given that there is only one observable, a squeeze is applied to get
+    # a 1-dimensional tensor
+    loss = mse_loss(model(values=x_train).squeeze(), y_train)
```

Comment:
do we want to add an `assert loss < 1e-3` at the end to make sure this will error if it does not converge in a future docs build?